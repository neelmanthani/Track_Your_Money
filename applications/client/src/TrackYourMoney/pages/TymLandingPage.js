import React from 'react'
import { Route, Routes, useNavigate } from 'react-router-dom'
import '.././css/tymLanding.css'
import TymHomePage from './TymHomePage'

function TymLandingPage() {
  const navigate = useNavigate();

  const handleLogin = () => {
    navigate('/login')
  }

  return (

    <>
      <div className='main-container-landing'>

        <section className='title'>
          <h3>Track Your Money</h3>
        </section>

        <section className='hero'>
          <h2> Take Control Of your money</h2>
          <p>Keep track of your small expenses to keep your financial life in track.</p>
        </section>

        {/* <section className="login_buttons"> */}
        <section className="main-button login_buttons">
          <button onClick={handleLogin}>Log In</button>
          <button onClick={() => navigate('/registration')}>Sign Up</button>
        </section>

      </div>


      <Routes>
        <Route path='/tymHome' element={<TymHomePage />}></Route>
        {/* <Route path='/registration' element={<TymRegistration />}></Route> */}
        {/* <Route path='/login' element={<TymLoginPage />}></Route> */}


      </Routes>
    </>
  )

}

export default TymLandingPage