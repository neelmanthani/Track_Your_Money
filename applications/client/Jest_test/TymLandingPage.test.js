/**
 * @jest-environment jsdom
 */
import renderer from 'react-test-renderer';
import TymLandingPage from '../src/TrackYourMoney/pages/TymLandingPage';
import {BrowserRouter as Router} from 'react-router-dom';


it('changes the class when hovered', () => {
  const component = renderer.create(
    <Router>
    <TymLandingPage></TymLandingPage>
    </Router>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});