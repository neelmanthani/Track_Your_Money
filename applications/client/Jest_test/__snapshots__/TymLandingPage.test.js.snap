// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`changes the class when hovered 1`] = `
<div
  className="main-container-landing"
>
  <section
    className="title"
  >
    <h3>
      Track Your Money
    </h3>
  </section>
  <section
    className="hero"
  >
    <h2>
       Take Control Of your money
    </h2>
    <p>
      Keep track of your small expenses to keep your financial life in track.
    </p>
  </section>
  <section
    className="main-button login_buttons"
  >
    <button
      onClick={[Function]}
    >
      Log In
    </button>
    <button
      onClick={[Function]}
    >
      Sign Up
    </button>
  </section>
</div>
`;
